Fix typos and grammar on README.md (#19)
courselab/snaskii21: README.md

 ## Playing the game
 

>  The game takes place on a rectangular arena where a snake continuously

Reviewer: I *think* this "on" should be "in"

```suggestion
 The game takes place in a rectangular arena where a snake continuously
```

Reviewer 2: Yeah the snake is technically **inside** the arena so I agree with "in"

 move in one of the four directions: left, right, up and down --- it never 

>  stops. As the snake moves, it looses energy, and if all this energy is exausted, the

Reviewer: Those comms are probably not necessary (as they would be in portuguese). Furthermore, "looses" should be "loses" and "exausted" should be "exhausted".

```suggestion
 stops. As the snake moves it loses energy and if all this energy is exhausted, the
```

Reviewer 2: I think we should let the second comma stay there. It makes the readability more fluid, since you are saying "energy" again (I'm not a grammar expert btw). What you think?
>stops. As the snake moves it loses energy, and if all this energy is exhausted, the

Reviewer: I'm not against it

>  snake dies. To recover energy, the snake needs to eat pieces of food which

Reviewer: ```suggestion
 snake dies. To recover energy, it needs to eat pieces of food, which
```

>  are constantly replaced at random positions. 

Reviewer: ```suggestion
 are constantly placed at random positions. 
```

Reviewer 2: Yeah makes sense. I don't think the food is replaced since a new one will be **placed** after the current is eaten (it's not implemented yet but I think it's how it'll work)

Be careful, though. The arena borders are electrified and would kill the snake
if touched. Morover, mind that the snake is poisonous and it would also die if 

> it accidently bites itself, i.e. if the head of snake crosses its own body (yes, 

Reviewer: I think it works better like this:

```suggestion
it accidently bites itself, i.e. if its head crosses its own body (yes, 
```

But if you'd rather keep it the other way, there's a "the" missing:
```suggestion
it accidently bites itself, i.e. if the head of the snake crosses its own body (yes, 
```

Reviewer 2: I liked the first option:

>it accidently bites itself, i.e. if it's head crosses it's own body (yes, 

But it's _its_ and not _it's_. There's no "is" there :P

Reviewer: > But it's _its_ and not _it's_. There's no "is" there :P

TIL you shouldn't use the apostrophe for the possessive pronoun

> this is weird for snakes, but this is a Tron Snake).

Reviewer: ```suggestion
this is weird for snakes, but it's a Tron Snake).
```
